components/fieldgroup/stories/fieldgroup.stories.js

```diff
@@ -2,10 +2,30 @@ import { Template } from "./template";
 
 import { default as Radio } from "@spectrum-css/radio/stories/radio.stories.js";
 
+/**
+ * A field group is a group of fields, usually radios (also known as a radio group) or checkboxes
+ * (also known as a checkbox group). A field group is composed of a field label, a group of radio
+ * inputs or checkboxes, and an optional help text component. The field label within the field group
+ * can be used to mark a field group as optional or required. The field group items other than the
+ * label must be wrapped in a nested `div` with `.spectrum-FieldGroupInputLayout` to control their
+ * layout separately from the label. Help text may or may not appear below a field group and is
+ * necessary when denoting invalid checkbox fields, invalid radio button fields, and required 
+ * fields. Invalid radio buttons and checkboxes are signified by negative help text.
```

Comment:
There two lines in the [docs site description](https://opensource.adobe.com/spectrum-css/fieldgroup.html) that said "Invalid radio buttons are signified only by negative Help text.. Invalid checkboxes are signified by negative Help text and the negative/invalid color on the input box." Meaning that an invalid radio group only gets negative (red) help text, but an invalid checkbox group gets negative (red) help text and also an invalid (red) color on the input box.

I combed through the internal docs for [Checkbox Group](https://spectrum.corp.adobe.com/page/checkbox-group/) and for [Radio](https://spectrum.corp.adobe.com/page/radio-group/) Group as well as the XD design specs for both of those components, and traced the addition of this documentation to #1486 and its [Jira issue](https://jira.corp.adobe.com/browse/CSS-107), and cannot find any evidence that the checkbox (but not the radio) should be red in this situation.

I'm honestly still on the fence about deleting it and think I'll try to follow up to see if there was any historical reasoning for this change that no one documented.